src/som/interpreter/Method.java

```diff
@@ -51,7 +51,7 @@ public Method(final String name, final SourceSection sourceSection,
     this.definition = definition;
     this.block = block;
     this.methodScope = methodScope;
-    assert methodScope.isFinalized();
+    //assert methodScope.isFinalized();
```

Comment:
if this doesn't hold anymore, tell me why, and remove the assertion, no commented out code in this repo!!!!!!